support-frontend/build.sbt

```diff
@@ -77,13 +77,17 @@ def getFiles(rootFile: File, deployName: String): Seq[(File, String)] = {
   getFiles0(rootFile)
 }
 
-Universal / javaOptions ++= Seq(
-  "-Dpidfile.path=/dev/null",
-  "-J-XX:MaxMetaspaceSize=256m",
-  "-J-XX:+PrintGCDetails",
-  s"-J-Xloggc:/var/log/${packageName.value}/gc.log",
+val jvmParameters = Def.setting(Seq(
+  "-XX:MaxMetaspaceSize=256m",
+  s"-Xlog:gc*:/var/log/${packageName.value}/gc.log", // https://docs.azul.com/prime/Unified-GC-Logging#enabling-unified-gc-logging
   "-XX:-OmitStackTraceInFastThrow",
+))
+val playParameters = Seq(
+  "-Dpidfile.path=/dev/null", // https://www.playframework.com/documentation/3.0.x/ProductionConfiguration#Changing-the-path-of-RUNNING_PID
 )
+// -J tells the packager to pass it through to the JVM
+// https://www.scala-sbt.org/sbt-native-packager/archetypes/java_app/customize.html#via-build-sbt
+Universal / javaOptions ++= playParameters ++ jvmParameters.value.map(param => "-J" + param)
```

Comment:
I have tweaked this structure because it wasn't clear that you have to prepend `-J` in order to supply JVM parameters.  This means that the `-XX:-OmitStackTraceInFastThrow` was being passed to play (which ignored it) rather than the JVM.  The -D argument is actually a play parameter, so it's in a separate list.

Reply:
I have tweaked this structure because it wasn't clear that [you have to prepend -J in order to supply JVM parameters](https://www.scala-sbt.org/sbt-native-packager/archetypes/java_app/customize.html#via-build-sbt). This means that the -XX:-OmitStackTraceInFastThrow was being passed to play (which actually seems to pass it on to the JVM anyway) rather than the JVM. The -D argument is actually a play parameter, so it's in a separate list.  But again, it does get passed to the JVM anyway.

I can tell this is (still) working by checking the command line to java.

<img width="667" alt="image" src="https://github.com/guardian/support-frontend/assets/7304387/b19c14ad-6cdd-4b49-bc30-e873e2c863a7">

Although as I say it was actually present before:
<img width="679" alt="image" src="https://github.com/guardian/support-frontend/assets/7304387/6608a031-06ef-4b3a-98eb-40bd5d93f4ed">